source/includes/fact-verifier-limitations.rst

```diff
@@ -0,0 +1,37 @@
+
+The embedded verifier has the following limitations:
```

Comment:
The embedded verifier also won't support using `buildIndexes:never` on a cluster that has collections with TTL indexes.
cc @mmcclimon

Reply:
@jddubois, @mmcclimon  We already note that the verifier doesn't check indexes and doesn't check collections with TTL indexes.  Is this a separate issue? Like it errors out on `buildIndexes: never`?

Reply:
Yes; sorry, this is a recent change and hasn't made it into the code yet, so it's not yet in the downstream changes document.

If mongosync is started with `buildIndexes: never` and the verifier sees a TTL collection, mongosync will return an error and the migration will fail.

Reply:
Fixed